### DISCUSSION TITLE
How to add text background feature

### DISCUSSION THREAD

**Discussion by bumpyy** [awcodes/filament-tiptap-editor#495] [Q&A]
Is there a way to add text background feature, every time I paste something from another website there's always background color (mostly white).

**Comment by awcodes** [comment]
I'm confused on what you are asking. Can you explain what you are expecting or wanting to happen?

**Reply by bumpyy** [reply]
So basically, when i copy content sometime the background will also get copied (as style)
![image](https://github.com/user-attachments/assets/94f7889f-ffb3-4366-a2d2-bb21a9e25ccd)
![image](https://github.com/user-attachments/assets/ff50d2c3-beb6-437b-9a5f-079ef115ddb3)

I think i need to write a custom extension, but i need some pointer.

**Reply by awcodes** [reply]
Not sure there's much that can be done, since the editor supports the style attribute. When you copy paste from places that also copy the html markup it will be honored when pasting.
